### DISCUSSION TITLE
Problem after Update -up and -ut

### DISCUSSION THREAD

**Discussion by mb01LINX** [projectdiscovery/nuclei#3707] [Q&A] [answered]
i try to update the nuclei template and nuclei it self by use -up and -ut buat after that the response is upnormal like this please see i try in wsl2 and windows is same error this ini windows  response line [←[33mWRN←[0m]
<img width="654" alt="image" src="https://github.com/projectdiscovery/nuclei/assets/106868598/15c0dce4-1ec2-4af4-b1fb-93cf81ecca67">

**Comment by ehsandeep** [comment, marked as answer]
@mb01LINX see https://github.com/projectdiscovery/nuclei/discussions/3648 and try using `nuclei -reset`

**Comment by mb01LINX** [comment]
Thank you, @ehsandeep  for helping me. The anomaly has been fixed.
